hoot-core/src/main/cpp/hoot/core/conflate/CookieCutter.cpp

```diff
@@ -92,7 +90,8 @@ void CookieCutter::cut(OsmMapPtr& cutterShapeOutlineMap, OsmMapPtr& doughMap)
   cutterShapeOutlineMap.reset();
 
   // remove the cookie cutter portion from the dough
-  MapCropper cropper(cutterShape);
+  MapCropper cropper;
+  cropper.setBounds(cutterShape);
```

Comment:
Using `setBounds` vs passing the bounds in from the constructor is more in line with how the rest of the `Boundable` implementations behave.